SDK/simVis/RadialLOSNode.cpp

```diff
@@ -146,13 +146,17 @@ void RadialLOSNode::setAzimuthalResolution(const Angle& value)
 }
 
 void RadialLOSNode::updateDataModel(const osgEarth::GeoExtent& extent,
-                               osg::Node*                 patch)
+                                    osg::Node*                 patch)
 {
-  if (getMapNode() && bound_.intersects(extent.getBoundingGeoCircle()))
+  if (getMapNode())
   {
-    if (los_.update(getMapNode(), extent, patch))
+    osgEarth::GeoCircle circle = extent.computeBoundingGeoCircle();
```

Comment:
Looks like an API break from the osgEarth-SDK-1.7 version.  We try to catch those per SDK release.  Internally we update only once every few weeks, or as needed.

`#if SDK_OSGEARTH_MIN_VERSION_REQUIRED(1,8,0)` should help.

I'll merge this in and take care of it.